shared-module/displayio/Display.c

```diff
@@ -80,6 +84,10 @@ void common_hal_displayio_display_construct(displayio_display_obj_t* self,
         bool delay = (data_size & DELAY) != 0;
         data_size &= ~DELAY;
         uint8_t *data = cmd + 2;
+        if (self->init_cs_toggle && self->set_cs != NULL) {
+            self->set_cs(self->bus, true);
+            self->set_cs(self->bus, false);
+        }
```

Comment:
I'm not sure this is going to be reliable, especially on the faster MCUs and with longer wires going to the display — the voltage might have not enough time to grow to TTL levels. I wonder if it wouldn't be better to toggle the CS pin at the start and at the end of the delay later in this loop.

Reply:
Basically leave the ``self->set_cs(self->bus, false)`` here, and move the ``self->set_cs(self->bus, true)`` to right after the send commands.

Reply:
The main reason I have the toggle commands right next to each other is because that’s how it functions in the Arduino driver.

I originally had it after the send command and it required adding an additional reset for the ST7789 display to function for it to work, which is the main reason I added this. When I moved it to before the send commands based on the comments by @ladyada, it worked much better. 

Also, I was testing with relatively long wires hooked up (about 6” or so) using an M4 processor and it worked consistently in this manner.

Reply:
After thinking about it for a bit, I could dd an arbitrary amount of delay between (1-2ms) between the commands if that would make you feel more comfortable.

Reply:
Thank you. It's initialization code, so it doesn't hurt us much if it takes a few ms longer.

Reply:
I looked in the datasheet to see about minimum CS toggle times, and the minimum times for CS setup are in the tens of nanoseconds, so it seems like this will not be an issue.

Reply:
Thank you, sorry for being paranoid.

Reply:
I think it was appropriate, and a small delay is good. Microcontrollers will only get faster.